src/solvers/truncation_parameters.jl

```diff
@@ -0,0 +1,14 @@
+default_maxdim() = typemax(Int)
+default_mindim() = 1
+default_cutoff() = 0.0
```

Comment:
```suggestion
default_cutoff(elt::Type{<:Real}) = zero(elt)
```
so that we don't assume double precision (`Float64`).